auth-web/src/components/auth/stepper/InfoStepper.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <div>
+    <v-container id="step-buttons-container">
+      <template v-for="(step, index) in steps">
+        <div class="step" :key="index">
+          <v-btn
+            fab
+            outlined
+            class="step__icon"
+            active-class="filled no-active"
+            :id=step.id
+            color="#1A5A96"
+            :to=step.to>
+            <v-icon>{{step.step}}</v-icon>
+          </v-btn>
+          <v-btn class="step__label" active-class="selected no-active" text color="#1A5A96" :to=step.to>
+            {{step.text}}
+          </v-btn>
+          <span :class="{ 'arrow-down': isCurrentStep(step) }"></span>
+        </div>
+      </template>
+    </v-container>
+    <div class="next-step-wrapper">
+      <span class="next-step-btn" :class="{ 'hide-next-btn': getCurrentStep() === steps.length }" @click="nextStep()">
+        <u>Next Step</u><v-icon color="#1A5A96">mdi-menu-right</v-icon>
+      </span>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Vue } from 'vue-property-decorator'
+
+@Component
+export default class Stepper extends Vue {
+  private steps: Array<any> = [
+    {
+      id: 'step-1-btn',
+      step: 1,
+      text: 'Decide on a Business Type',
+      to: '/home-dev/decide-business-info'
+    },
+    {
+      id: 'step-2-btn',
+      step: 2,
+      text: 'Request a Name',
+      to: '/home-dev/request-name-info'
+    },
+    {
+      id: 'step-3-btn',
+      step: 3,
+      text: 'Incorporate or Register',
+      to: '/home-dev/incorp-or-register-info'
+    },
+    {
+      id: 'step-4-btn',
+      step: 4,
+      text: 'Maintain Your Business',
+      to: '/home-dev/maintain-business-info'
+    }
+  ]
+
+  private nextStep (): void {
+    const currentStepIndex = this.getCurrentStep()
+    const nextStep = this.steps[currentStepIndex]
+    this.$router.push(nextStep.to)
+  }
+
+  private isCurrentStep (step: any): boolean {
+    return this.$route.path === step.to
+  }
+
+  private getCurrentStep (): number {
+    const route = this.$route.path
+    for (const path of this.steps) {
+      if (path.to === route) {
+        return path.step
+      }
+    }
+  }
```

Comment:
This doesn't return anything if route doesn't match a path. Return -1?

Reply:
Updated 👍